ci/travis/bazel_detect_flake_options

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
+# On the Linux master build, retry core integration tests 3x to detect flakes with higher fidelity.
+
+if [ "${BUILDKITE_PULL_REQUEST}" = "false" ]; then
+    echo "--runs_per_test=//python/ray/tests.*@3 --runs_per_test_detects_flakes --experimental_cancel_concurrent_tests --flaky_test_attempts=1"
```

Comment:
Does `--flaky_test_attempts=1` mean we try 3 times (`--runs_per_test=//python/ray/tests.*@3`) and run one more time if it fails?